.github/workflows/ci.yml

```diff
@@ -178,6 +192,20 @@ jobs:
             cmdArgs: --plugin-name='no-backend-scenes' --org-name='myorg' --plugin-type='scenesapp' --no-backend
             hasBackend: false
     steps:
+      - id: get-secrets
```

Comment:
not 100% sure if we can combine these two so I left it in two steps.

Reply:
I think it's possible to combine (see [example](https://github.com/grafana/shared-workflows/tree/main/actions/get-vault-secrets#using-environment-variables-default)) but fine to leave them separated for now as long as they work - the steps just take longer.